home/home3.1/InverseTest.java

```diff
@@ -0,0 +1,37 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class InverseTest {
+
+    @Test
+    public void PasitiveArguments() {
+        Inverse inverse = new Inverse();
+        double result = inverse.calculateInverse(2, 3, 4);
```

Comment:
it is better to split visually blocks for build , operate check 
also we should avoid formulas inside unit tests so we can compared to calculated value. Formula can be changed later with the same green test. In your case you hardcoded that values to messages for asserts. That messages are useless as in case of failure developer will see both operands and in any case will analyse the code to find the root cause.